src/auth/auth.service.ts

```diff
@@ -301,12 +305,37 @@ export class AuthService {
   }
 
   deleteUser = async (email: string) => {
-    const resp = await this.userRepo.delete({ email });
-    if (resp.affected == 0) {
+    try {
+      const user = await this.userRepo.findOne({
+        where: { email },
+      });
+
+      if (!user) {
+        throw new BadRequestException(
+          ZaLaResponse.BadRequest(
+            'User Not found',
+            'The Email Does not exists',
+            '400',
+          ),
+        );
+      }
+      //  send Delete request to the profile service to delete the user profile
+      // Axios
+      const coreUrl = `${this.configService.get<string>(
+        configConstant.baseUrls.coreService,
+      )}/profile/${user.profileID}`;
```

Comment:
The user.profileID here is it identical to the id column in profile in core service? If they are not, this will not work. Please verify this. I think you should be using the userId

Reply:
It's identical to the Id of the profile

During user OTP verification when the profile is created the profile.id is assigned to user.profileId